docs/using-ngrok-with/k8s.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Kubernetes
+---
+
+# Using ngrok with Kubernetes 
+------------
+
+## Introduction
+
+The [ngrok Ingress Controller for Kubernetes](https://github.com/ngrok/kubernetes-ingress-controller) is our official open-source controller for adding public and secure ingress traffic to your k8s clusters. If you’ve used ngrok in the past, you can think of the ngrok Ingress Controller as ngrok packaged an idiomatic k8s resource — available as helm chart, configurable via manifest (using the `kind: Ingress` construct), and compatible with k8s best practices. 
+
+In this tutorial, you will install the ngrok Ingress Controller and run a sample 2048 app with public ingress access and security provided by ngrok.
+
+:::caution This tutorial requires:
+1. A local environment with Kubernetes installed and configured. For this tutorial, we will use `k3d`, `kubectl`, and `helm`.
+1. An [ngrok account](https://ngrok.com/signup) with a Pro subscription. The subscription is required in order to configure the ngrok Ingress Controller with a custom domain and an Edge Configuration.
+:::
+
+## Get started with the ngrok Ingress Controller for Kubernetes
+
+## Step 1: Get your ngrok authtoken and API key
+
+:::tip
+The ngrok Ingress Controller requires:
+- The authtoken to launch connections from ngrok's points of presence to your k8s cluster.
```

Comment:
do we normally talk about points of presence or just "the edge servers"?
I'm also not sure which way we normally say its connecting, but i envision it as connecting from the ingress controller in your k8s cluster, to the edge servers to allow traffic to back down the other way

Reply:
changed the copy. Feel free to make suggestions.